base-noprelude.cabal

```diff
@@ -251,14 +261,36 @@ library
       , Type.Reflection
       , Type.Reflection.Unsafe
       , Unsafe.Coerce
+      -- TODO: remove
```

Comment:
ah, I see that this comes from the `cabal` file for `base`, but I think that we can remove this comment

Reply:
Nitpick: I'm un-resolving this simply because I see that there's now an empty line (with spaces).

I'd prefer that to be no line at all, as it seems to be the case for previous commits :slightly_smiling_face:

Reply:
Ooook; dropped.